server.go

```diff
@@ -1587,14 +1589,28 @@ func (s *server) peerConnected(conn net.Conn, connReq *connmgr.ConnReq,
 // utilize the ordering of the local and remote public key. If we didn't use
 // such a tie breaker, then we risk _both_ connections erroneously being
 // dropped.
-func shouldDropLocalConnection(local, remote *btcec.PublicKey) bool {
+func (s *server) shouldDropLocalConnection(local, remote *btcec.PublicKey) bool {
 	localPubBytes := local.SerializeCompressed()
 	remotePubPbytes := remote.SerializeCompressed()
 
 	// The connection that comes from the node with a "smaller" pubkey
 	// should be kept. Therefore, if our pubkey is "greater" than theirs, we
 	// should drop our established connection.
-	return bytes.Compare(localPubBytes, remotePubPbytes) > 0
+	supposedToDrop := bytes.Compare(localPubBytes, remotePubPbytes) > 0
```

Comment:
`godoc` comment for the method hasn't been updated.

Reply:
👍